cli/ballerina-cli/src/test/java/io/ballerina/cli/cmd/NewCommandTest.java

```diff
@@ -453,6 +454,51 @@ public void testNewCommandWithLib() throws IOException {
         Assert.assertTrue(readOutput().contains("Created new package"));
     }
 
+    @Test(description = "Test new command with tool template")
+    public void testNewCommandWithTool() throws IOException {
+        System.setProperty(USER_NAME, "testuserorg");
+        Path packageDir = tmpDir.resolve("tool_sample");
+        String[] args = {packageDir.toString(), "-t", "tool"};
+        NewCommand newCommand = new NewCommand(printStream, false);
+        new CommandLine(newCommand).parseArgs(args);
+        newCommand.execute();
+        // Check with spec
+        // project_name/
+        // - Ballerina.toml
+        // - BalTool.toml
+        // - Package.md
+        // - tool
+        // - .gitignore       <- git ignore file
+
+        Assert.assertTrue(Files.exists(packageDir));
+        Assert.assertTrue(Files.isDirectory(packageDir));
+        Assert.assertTrue(Files.exists(packageDir.resolve(ProjectConstants.BALLERINA_TOML)));
+        String packageName = Paths.get(args[0]).getFileName().toString();
+        String tomlContent = Files.readString(
+                packageDir.resolve(ProjectConstants.BALLERINA_TOML), StandardCharsets.UTF_8);
+        String expectedContent = "[package]\n" +
+                "org = \"testuserorg\"\n" +
```

Comment:
This `\n` will identify as new line in ubuntu, is it work same for windows as expected?

Reply:
In windows, I think it is `\r\n`.

Reply:
This is a toml file output hence this should not be a problem

Reply:
ok